docs/sidebar/sidebar-items.md

```
@@ -3,83 +3,100 @@ aside: false
+<script setup>
+import SandboxIframe from '../.vitepress/theme/components/sandbox/SandboxIframe.vue'
+</script>
+
 # Sidebar Items
 
-To automatically generate sidebar items, you can use the `useSidebar` function. Configure your `.vitepress/config.js` as follows:
+The `useSidebar` composable provides functions to generate sidebar items based on the OpenAPI specification. You can generate sidebar items by operations grouped by tags, by tags, or by paths.
+
+## Operations grouped by tags
+
+<div class="grid grid-cols-3 gap-4">
```

Markdownlint (0.37.0)

Element: div
Inline HTML

(MD033, no-inline-html)